src/modules/Builder/features/ActivityItems/ItemConfiguration/Settings/ItemSettingsController/ItemSettingsGroup/ItemSettingsGroup.tsx

```diff
@@ -94,18 +111,18 @@ export const ItemSettingsGroup = ({
                   control={
                     <Checkbox
                       name={settingKey}
-                      checked={value?.includes(settingKey)}
+                      checked={!!get(value, settingKey) || false}
```

Comment:
`!!` already casts to a boolean value, no need `|| false`, right?

Reply:
fixed